client/app/billing/orders/billing-orders.controller.js

```diff
@@ -1,144 +1,63 @@
-angular.module('Billing.controllers').controller('Billing.controllers.Orders', ($q, $log, $scope, $location, $translate, Alerter, BillingOrders, BillingOrdersApiv7, BillingOrderStatusEnum, BillingOrdersStatusFilters, BillingUser) => {
-  $scope.itemsPerPage = 10;
-  $scope.orderIds = [];
-  $scope.loaders = {
-    orders: false,
-  };
-  $scope.activeFilter = {};
-  $scope.ordersStatus = {};
-
-  function init() {
-    BillingOrdersStatusFilters.getFilterConfig()
-      .then((filterConfig) => {
-        $scope.statusFilters = filterConfig;
-        const search = $location.search();
-        if (search.status) {
-          $scope.activeFilter.status = _.find($scope.statusFilters, {
-            id: search.status,
-          });
-        }
-        if (search.orderId) {
-          $scope.activeFilter.orderId = parseInt(search.orderId, 10);
-        }
+import _ from 'lodash';
+
+export default class BillingOrdersCtrl {
+  /* @ngInject */
+
+  constructor($q, $log, $state, $translate, OvhApiMe,
+    OvhApiMeOrder, constants, orders, schema, criteria, filter) {
+    this.$q = $q;
+    this.$log = $log;
+    this.$state = $state;
+    this.$translate = $translate;
+    this.OvhApiMe = OvhApiMe;
+    this.OvhApiMeOrder = OvhApiMeOrder;
+    this.orders = orders;
+    this.schema = schema;
+    this.criteria = criteria || [];
+    this.filter = filter;
+    this.getSubsidiary().then((subsidiary) => {
+      this.billingGuideUrl = constants.urls[subsidiary].guides.billing;
+    });
+  }
 
-        if (angular.isUndefined($scope.activeFilter.status)) {
-          $scope.activeFilter.status = _.first($scope.statusFilters);
-        }
+  getSubsidiary() {
+    return this.OvhApiMe.v6().get().$promise.then(({ ovhSubsidiary }) => ovhSubsidiary);
```

Comment:
The /me payload is injected in the `app.routes.js` as `currentUser`. Maybe you could injetc it directly instead of refetching the info